### PR TITLE
Operator: recoverable tokens

### PR DESCRIPTION
**Pull request by alexroan**

### CODE REVIEW

File: evm-contracts/src/v0.7/dev/Operator.sol [hunk 1]
```diff
@@ -269,6 +272,33 @@ contract Operator is
...
+    hasRecoverableFunds(amount)
+  {
+    linkToken.transfer(recipient, amount);
+  }
+
+  /**
+   * @notice Returns the amount of LINK that is recoverable (mistakenly sent to this contract)
+   * @return uint256 recoverable token amount
+   */
+  function recoverable()
```
Comment (se3000) [inline evm-contracts/src/v0.7/dev/Operator.sol:293]: Do you have a scenario in mind to keep `withdrawable` and `recoverable` separate? I vote to consolidate them. I think they're close enough, and just presenting `balance - s_nonWithdrawableTokens` is all I can see people caring about. Plus, we save contract space.

**Review by se3000 — APPROVED** [review]

**Review by se3000 — CHANGES_REQUESTED** [review]
Meant to request changes with those last comments.

File: evm-contracts/src/v0.7/dev/Operator.sol (hunk 1 above)
Comment (alexroan) [reply evm-contracts/src/v0.7/dev/Operator.sol:293]: Nice, yeah that would also negate the extra gas costs associated with maintaining both.

File: evm-contracts/src/v0.7/dev/Operator.sol [hunk 2]
```diff
@@ -402,6 +403,15 @@ contract Operator is
     ));
   }
 
+  /**
+   * @notice Returns the LINK available in this contract, not locked in escrow
+   * @return uint256 LINK tokens available
+   */
+  function fundsAvailable() internal view returns (uint256) {
```
Comment (se3000) [inline evm-contracts/src/v0.7/dev/Operator.sol:410]:
```suggestion
  function fundsAvailable() private view returns (uint256) {
```
I think we should default to private by default and only make things internal or public when we need to.

**Review by se3000 — APPROVED** [review]

**Pull request merged**